### DISCUSSION TITLE
Error on transfer-02-consumer-pull samples

### DISCUSSION THREAD

**Discussion by cgx-avenue** [eclipse-edc/Samples#188] [Q&A] [answered]
Hi all,
I followed the tutorial of transfer folder and failed to get the file transferred during transfer-02-consumer-pull sample. I checked the results from provider terminal output like below:
![2024-01-02_14-16](https://github.com/eclipse-edc/Connector/assets/10353881/0f6dba67-0256-46b2-9634-73a85e7e6450)
I modified corresponding contract agreement id and fulfilled all the requirements in transfer-00 and transfer-01 samples. Could anyone help me on this? Where should I modify to make it successful?
Thanks.

**Comment by ndr-brt** [comment, marked as answer]
It looks like you missed to register the data plane ([step 3 of the prerequisites](https://github.com/eclipse-edc/Samples/tree/main/transfer/transfer-00-prerequisites#3-register-data-plane-instance-for-provider))
